source/views/news/index.tsx

```diff
@@ -1,57 +1,66 @@
 import * as React from 'react'
-import {TabNavigator, TabBarIcon} from '@frogpond/navigation-tabs'
-
-import * as newsImages from '../../../images/news-sources'
-import NewsContainer from './news-container'
-
-const NewsView = TabNavigator({
-	StOlafNewsView: {
-		screen: ({navigation}) => (
-			<NewsContainer
-				navigation={navigation}
-				source="stolaf"
-				thumbnail={newsImages.stolaf}
-				title="St. Olaf"
+
+import {TabBarIcon} from '@frogpond/navigation-tabs'
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs'
+import {NativeStackNavigationOptions} from '@react-navigation/native-stack'
+
+import * as newsImages from '../../../images/news-sources/index'
+import {NewsList} from './news-list'
+
+type Params = {
+	StOlafNewsView: undefined
```

Comment:
Should these really all be typed as `undefined`?

Reply:
They should not, no. We currently have that pattern across the app, though. I hope to shore up our types with #6044.